assignments/prerequisites/python-programming-introduction.ipynb

```diff
@@ -1,113 +1,88 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    class Solution:
</pre><p>Why you need the class Solution?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/344/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>